Makefile

```diff
@@ -93,16 +99,34 @@ clean_deployments: ## Clean all deployments stores
 	@echo "[OTOPS] Cleaning all deployments stores"
 	@rm -rf ${OTOPS_PATH_DEPLOYMENT}
 
+clean_clickhouse_release: ## Clean the Clickhouse data image
+	@echo "[OTOPS] Cleaning the Clickhouse data image"
+	$(eval include .env)
+	@cd $(shell dirname ${OTOPS_CLEAN_DATA}) && ./$(shell basename ${OTOPS_CLEAN_DATA}) clickhouse
+
+clean_elastic_search_release: ## Clean the Elastic search data image
+	@echo "[OTOPS] Cleaning the Elastic search data image"
+	$(eval include .env)
+	@cd $(shell dirname ${OTOPS_CLEAN_DATA}) && ./$(shell basename ${OTOPS_CLEAN_DATA}) elastic
+
+clean_webapp_release: ## Clean the Open Targets Platform Webapp release bundle
+	@echo "[OTOPS] Cleaning Open Targets Platform Webapp release bundle"
+	$(eval include .env)
+	@cd $(shell dirname ${OTOPS_CLEAN_DATA}) && ./$(shell basename ${OTOPS_CLEAN_DATA}) webapp
+
+clean_release: clean_clickhouse_release clean_elastic_search_release clean_webapp_release ## Clean all data images and webapp bundle
```

Comment:
You also need to remove the _release_ folder, otherwise the targets related to downloading the files will not be run by the Makefile